src/main/resources/META-INF/rewrite/spring-boot-30.yml

```diff
@@ -33,7 +33,7 @@ recipeList:
   - org.openrewrite.java.spring.boot3.LegacyJmxExposure
   - org.openrewrite.java.spring.boot3.ActuatorEndpointSanitization
   - org.openrewrite.java.spring.boot3.Saml
-  - org.openrewrite.java.spring.boot3.SpringBootProperties_3_0_0
+  - org.openrewrite.java.spring.boot3.SpringBootProperties_3_0
```

Comment:
See comment below, we cannot make this change to the recipe name itself (without possibly breaking downstream users that are directly calling this recipe)

Reply:
Sounds good - revert this change